### PR TITLE
BLUE-210: Update tooltips to be WCAG compliant with Pharos tooltips

### PR DESCRIPTION
**Pull request by Niznikr**
Resolves BLUE-210

## Description

In order to be compliant with WCAG 2.1 standards for [tooltips](https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html), the `ng-bootstrap` tooltips have been replaced with [Pharos tooltips](http://pharos-storybooks.apps.test.cirrostratus.org/wc/?path=/docs/tooltip--base). Left `ang-promo-tooltip` alone as those seemed fine.

## Notes

- Most of these were a quick replace. The viewer icon buttons stylings from Bootstrap don't affect the shadow DOM so there is a minor UX difference with the tooltips added. Curious if that is a deal breaker for folks.
- Also updated the favicon as requested in BLUE-44
- Looking for guidance if unit/integration tests need updating and how to go about that

## Checks

**Have you written any necessary unit tests?**

- [ ] Yes
- [ ] Not yet
- [ ] Not applicable

**Have you added or updated any necessary integration tests?**

- [ ] Yes
- [ ] Not yet
- [ ] Not applicable

**Browsers tested on:**

- [ ] Chrome
  - [ ] Mobile
- [ ] Safari
  - [ ] Mobile
- [ ] Firefox
  - [ ] Mobile
- [ ] Edge
- [ ] IE11
- [ ] Not applicable

### CODE REVIEW

**Comment by Niznikr** [comment]
Quick question! Will investigate to see what browsers Artstor says they support.

File: src/app/asset-page/artstor-viewer/artstor-viewer.component.scss [hunk 1]
```diff
@@ -83,7 +80,22 @@ $desktop: 992px;*/
...
 
-
+    .btn--icon {
+        border-radius: 0;
+    }
+    // When remove from comparison, or present icon is by itself and needs a full border-radius
+    .btn--icon.standalone-remove-icon, .btn--icon.standalone-present-icon {
+        border-radius: 4px;
+    }
+    pharos-tooltip:first-of-type .btn--icon {
```
Comment (daneah) [inline src/app/asset-page/artstor-viewer/artstor-viewer.component.scss:90]: Would it be more understandable later if you added e.g. `.asset-viewer__buttons__button` (yuck) to each tooltip so you can select like this?

```suggestion
   .asset-viewer__buttons__button:first-child .btn--icon {
       ...
   }
```

Comment (Niznikr) [reply src/app/asset-page/artstor-viewer/artstor-viewer.component.scss:90]: Not a bad idea!

**Comment by Niznikr** [comment]
@michael-iden Looks like they do support IE11. Added the polyfills.

**Pull request merged**